crates/swc_ecma_transforms_module/src/common_js.rs

```diff
@@ -213,6 +213,18 @@ impl Fold for CommonJs {
                     for ident in found {
                         exports.push(ident.sym.clone());
                     }
+
+                    if let Decl::Var(VarDecl { decls, .. }) = decl {
+                        for var_decl in decls {
+                            if let VarDeclarator {
```

Comment:
I think we should use `find_pat_ids` here instead.

Reply:
Can you add a test case for exporting something with an array pattern or an object pattern?

Reply:
ok